packages/recomponents/src/components/r-tabs/r-tabs.scss

```diff
@@ -28,6 +28,7 @@
 }
 
 .r-tab-item .r-tab-link {
+    @include reset-button;
```

Comment:
nice cleanup on this file. does recomponents not currently have normalization for CSS?

Reply:
Looks like there is currently no CSS normalization. (Well to be exact, none except for this single rule: [packages/recomponents/src/styles/theme.scss#L28-L29](https://github.com/Rebilly/rebilly-recomponents/blob/master/packages/recomponents/src/styles/theme.scss#L28-L29))
Perhaps we should discuss our startegy for CSS normalization vs reset, and settle on an approach

Reply:
I think we should stick with CSS reset/nornalization mixins, in order to avoid such rules bleeding into the global scope

Reply:
A universal namespacing class is a great idea, I like it ^_^ 
Let's get some more opinions on this on slack